qt_ui/widgets/QTopPanel.py

```diff
@@ -127,15 +130,22 @@ def setGame(self, game: Optional[Game]):
         self.budgetBox.setGame(game)
         self.factionsInfos.setGame(game)
 
-        self.passTurnButton.setEnabled(True)
-        if game and game.turn > 0:
-            self.passTurnButton.setText("Pass Turn")
+        state = game.check_win_loss()
```

Comment:
Locking controls in QTopPanel, this is one of the things I'm considering to refactor to make the code a little cleaner...